src/core/include/ngraph/variant.hpp

```diff
@@ -47,9 +47,17 @@ template <typename VT>
 class OPENVINO_DEPRECATED("Please use ov::Any to store VT directly") VariantWrapper {};
 
 OPENVINO_SUPPRESS_DEPRECATED_START
+#if defined(_MSC_VER)
+#    pragma warning(push)
+// '__declspec(dllexport)' and 'extern' are incompatible on an explicit instantiation
+#    pragma warning(disable : 4910)
+#endif
 extern template class OPENVINO_API VariantImpl<std::string>;
 extern template class OPENVINO_API VariantImpl<int64_t>;
```

Comment:
maybe the compiler is right and we need to move this instantiation to cpp?

Reply:
I am not sure that it will affect nothing. So first of all I would like to fix CI, and after that I will apply comments.